cabal.project

```diff
@@ -13,8 +13,8 @@ repository cardano-haskell-packages
 -- See CONTRIBUTING for information about these, including some Nix commands
 -- you need to run if you change them
 index-state:
-  , hackage.haskell.org 2024-01-16T14:48:13Z
-  , cardano-haskell-packages 2024-01-18T16:26:04Z
```

Comment:
Out of curiosity, why did you bump CHaP? It's not strictly necessary in this PR right?

Reply:
No it's not. In general it's good to have similar versions of hackage and chap (chap > hackage ideally), because chap packages were built against quite recent hackage, so it just gives us bigger probability of chap packages compatibility.